src/create-twilio-function.js

```diff
@@ -52,14 +53,19 @@ async function createTwilioFunction(config) {
   // Scaffold project
   const spinner = ora();
   spinner.start('Creating project directories and files');
-  // await createDirectory(projectDir, 'functions');
-  // await createDirectory(projectDir, 'assets');
+
   await createEnvFile(projectDir, {
     accountSid: config.accountSid,
     authToken: config.authToken
   });
   await createNvmrcFile(projectDir);
-  await createExampleFromTemplates(projectDir);
+  if (config.template) {
+    await createDirectory(projectDir, 'functions');
+    await createDirectory(projectDir, 'assets');
+    await downloadTemplate(config.template, '', projectDir);
```

Comment:
This is currently crashing and burning if you enter a non-existing template.

Reply:
I've been meaning to revisit error handling for everything past the original directory discovery. Probably need to actually handle them...

Reply:
Ok, added error handling and refined the way it's all displayed using `ora`.